docs/features/software-catalog/well-known-annotations.md

```diff
@@ -40,6 +40,23 @@ expecting a two-item array out of it. The format of the target part is
 type-dependent and could conceivably even be an empty string, but the separator
 colon is always present.
 
+### backstage.io/managed-by-origin-location
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    backstage.io/managed-by-origin-location: github:http://github.com/backstage/backstage/catalog-info.yaml
```

Comment:
```suggestion
    backstage.io/managed-by-origin-location: url:http://github.com/backstage/backstage/catalog-info.yaml
```

...not sure, but should this prefix be URL? if not, ignore this feedback!

Reply:
It should!

Reply:
This is wrong in the whole file. I'll fix it.